test/profile/test-storage.js

```diff
@@ -16,7 +16,7 @@ import { ProfileStorage, VisitType } from '../../app/services/storage';
 
 describe('Utility tests', () => {
   it('Compares sets correctly.', (done) => {
-    expect(Immutable.Set(['a', 'b']).equals(Immutable.Set(['b', 'a'])));
+    expect(Immutable.Set(['a', 'b']).equals(Immutable.Set(['b', 'a']))).toBe(true);
```

Comment:
Correct. `be` is `===`. Note that we use `.equals(…).toBe(true)` here because I haven't yet sent in a patch for `expect.js` to do the right thing with sets. I could use `.toBeTruthy` instead of `toBe(true)`, but this is more precise.